docs/monitoring/tag-aliases-support.md

```diff
@@ -0,0 +1,107 @@
+title: Tag Aliases
+description: Tag aliases are name-value pairs used to identify a group of tag values i.e. alias for a group of tags. For example, user can create a tag alias `env:prod` which identifies a set of hosts. 
+
+## What are Tag Aliases?
+
+Tag aliases are name-value pairs used to identify a group of tag values i.e. alias for a group of tags. For example, user can create a tag alias `env:prod` which identifies a set of hosts running in prod setup. So user can define this tag alias and map it set of `os.host` tag values. Tag aliases can be defined from UI, configured in agent or automatically sent by agents.
+
+## Automatic Tags Aliases from agent
+
+The Infra Agent automatically collects following tag aliases and sends them periodically to Sematext Cloud. These tag aliases are purged periodically. It is not recommended to use these names in configured/user-defined tag aliases.
+
+### Cloud metadata
+
+The cloud metadata from AWS, Azure and GCE instances is collected as tag aliases. These tag aliases are mapped to `os.host` tag.
+
+| Name  | Tag Name  | Supported Cloud Providers  |
+|:--|:--|:--|
+|  Provider Type |  cloud.type |  AWS, GCE, Azure |
+|  Instance Identifier |  cloud.instance.id |  AWS, GCE, Azure |
+|  Instance Name |  cloud.instance.name |  Azure, GCE |
+|  Instance Type |  cloud.instance.type |  AWS, GCE, Azure |
+|  Region |  cloud.region |  AWS, Azure |
+|  Availability Zone |  cloud.zone |  AWS, GCE |
+|  Project Identifier |  cloud.project |  GCE |
+|  User-defined tags |  - |  AWS, GCE, Azure |
+
+To collect user-defined tags you need to define the IAM roles listed below:
```

Comment:
To be clearer, maybe use "aws tags" here instead of just "tags"?

Reply:
Changed it to `cloud tags`